src/vivarium/framework/results/manager.py

```diff
@@ -71,36 +70,36 @@ def setup(self, builder: "Builder"):
 
         builder.value.register_value_modifier("metrics", self.get_results)
 
-    def on_post_setup(self, event: Event):
-        # update self._metrics to have all output keys
-        def create_measure_specific_keys(measure: str, stratifications: List[str]) -> None:
-            measure_str = f"MEASURE_{measure}"
-            individual_stratification_strings = [
-                [
-                    f"{stratification.name.upper()}_{category}"
-                    for category in stratification.categories
-                ]
-                for stratification in sorted(
```

Comment:
I remember there was a reason for this sorting and calling the upper() method on the stratification name but I can't remember why now. I think if we see what we expect in the index of the initialized dataframes we should be go to go, or will catch the reasons for doing this further along in the process.

Reply:
Yeah I wasn't sure about the sorting. the upper I think was just to make the columns easier to parse later. I agree that we'll find out down the line!